tests/unit/pre-loader-specs.ts

```diff
@@ -3,18 +3,18 @@ import {expect} from "chai";
 
 describe("PreLoader animations", () => {
     const element = document.createElement("div");
-    element.setAttribute("id", "pre-loader");
+    element.setAttribute("id", "preloader");
 
-    const preLoader = new PreLoader(element);
+    const preLoader = new PreLoader();
 
     it("should add `loaded` class on hiding", async () => {
         await preLoader.hide();
-        expect(element.outerHTML).to.equal('<div id="pre-loader" class="loaded"></div>');
+        expect(document.body.className).to.equal("loaded");
```

Comment:
To equal?
What if another library will add another class.
`document.body.classList.contains("loaded")` must be used